.github/workflows/publish-containers.yml

```
@@ -37,46 +51,54 @@ permissions:
   build-and-publish:
-    name: OCI Image (${{ matrix.target_family }} / ${{ matrix.amdgpu_targets }})
+    name: OCI Image (${{ matrix.flavor }} / ${{ matrix.amdgpu_targets }})
     runs-on: ubuntu-24.04
-    timeout-minutes: 60
+    timeout-minutes: 30
+    # T7: consume the prebuilt artifact from the build workflow instead of
+    # re-cloning the engine and re-running cmake. The build workflow
+    # produces a single .tar.gz per matrix leg with a SHA-suffixed name;
+    # we fetch the one matching `matrix.flavor` and reuse it as the
+    # build context for the publish-time thin Containerfile.
+    needs: build-heretek-rocmfpx.yml
```

actionlint (1.7.12)

[error] invalid job ID "build-heretek-rocmfpx.yml". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)